src/Tests/Violation/ViolationChecker/InterfaceViolationCheckerTest.php

```diff
@@ -32,7 +32,7 @@ public function testCheck()
         $deprecatedInterfaceUsage = $deprecatedInterfaceUsage->reveal();
 
         $ruleSet = $this->prophesize('SensioLabs\DeprecationDetector\RuleSet\RuleSet');
-        $file = $this->prophesize('Symfony\Component\Finder\SplFileInfo')->reveal();
+        $this->prophesize('Symfony\Component\Finder\SplFileInfo')->reveal();
```

Comment:
This line could be remove completely, if $file is not used anywhere in the test, then it makes no sense creating the mock.